subsys/bluetooth/mesh/brg_cfg.c

```diff
@@ -43,7 +43,7 @@ static void brg_tbl_compact(void)
 			j++;
 		}
 	}
-	memset(&brg_tbl[j], 0, sizeof(brg_tbl[j]));
+
 	bt_mesh_brg_cfg_row_cnt--;
```

Comment:
How does it work if multiple elements are removed from the table?
It is possible here: https://github.com/zephyrproject-rtos/zephyr/blob/2468f21e41d63d0daf7b9816d8d0508ba7863f53/subsys/bluetooth/mesh/brg_cfg.c#L335-L349 if I'm not mistaken.
Compact functionality assumes there is only sinle line that should be compacted.

Reply:
And if it uses `bt_mesh_brg_cfg_row_cnt`, why does it check `direction` field at line 337...